lizzy_client/metrics.py

```diff
@@ -28,7 +28,7 @@ def report_metric(metric_name: str, value: int, fail_silently: bool=True):
     try:
         lizzy_domain = urlparse(configuration.lizzy_url).netloc
         lizzy_name, _ = lizzy_domain.split('.', 1)
-    except:
+    except Exception as e:
```

Comment:
`as e` is redundant since we don't use it.

Reply:
I've added it because otherwise flake complains with `lizzy_client/metrics.py:31:5: E722 do not use bare except`. What would you suggest?

Reply:
`except Exception:` is enough to solve the issue.